lib/hanami/view/rendering.rb

```diff
@@ -104,6 +105,25 @@ def render
           layout.render
         end
 
+        # It tries to invoke a method for the view or a local for the given key.
+        # If the lookup fails, it returns a null object.
+        #
+        # @return [Objeect,Hanami::View::Rendering::NullLocal] the returning value
+        #
+        # @since x.x.x
+        #
+        # @example
+        #   <% if local(:plan).overdue? %>
+        #     <h2>Your plan is overdue.</h2>
+        #   <% end %>
+        def local(key)
```

Comment:
What do you think can we remove or replace duplicate logic to `Rendering#local`?